kani-compiler/src/codegen_cprover_gotoc/codegen/intrinsic.rs

```diff
@@ -445,7 +446,17 @@ impl<'tcx> GotocCtx<'tcx> {
             "cosf64" => codegen_simple_intrinsic!(Cos),
             "ctlz" => codegen_count_intrinsic!(ctlz, true),
             "ctlz_nonzero" => codegen_count_intrinsic!(ctlz, false),
-            "ctpop" => self.codegen_expr_to_place(p, fargs.remove(0).popcount()),
+            "ctpop" => {
```

Comment:
For readability, can you please move this to a little function?

Reply:
Done